### DISCUSSION TITLE
Hello, I want to use YOLOv8 to detect the images collected by multiple cameras at the same time, how to operate, thank you

### DISCUSSION THREAD

**Discussion by HYSzuibai** [ultralytics/ultralytics#7492] [Q&A]
**Hello, I want to use YOLOv8 to detect the images collected by multiple cameras at the same time, how to operate, thank you**

**Comment by pderrenger** [comment]
@HYSzuibai hello! To detect images from multiple cameras simultaneously with YOLOv8, you can use the `predict` mode with multiple sources. You'll need to load your model and then call the `predict` method with a list of sources corresponding to your camera feeds. Here's a concise example:

```python
from ultralytics import YOLO

# Load your pretrained model
model = YOLO('yolov8n.pt')  # Replace with your model of choice

# List of camera sources (replace with your camera IPs or device IDs)
# sources.streams with lines '0', 'http://192.168.0.101:8080/video', 'http://192.168.0.101:8080/video2'

# Perform detection on all camera feeds
results = model.predict(source=sources.streams)

# Process results for each camera feed
for result in results:
    # Handle detection results here
    print(result.pandas().xyxy)
```

Make sure to replace `'yolov8n.pt'` with the path to your actual model file and `camera_sources` with the correct sources for your cameras. If you're using IP cameras, replace the example URLs with the actual stream URLs of your cameras. For USB/webcams, use the device IDs (e.g., `'0'`, `'1'`, etc.).

For more details on handling the results and customizing the detection process, please refer to the [Predict documentation](https://docs.ultralytics.com/modes/predict). 🚀
